thirdparty/github.go

```diff
@@ -35,6 +35,9 @@ const (
 	githubWrite      = "write"
 
 	GithubInvestigation = "Github API Limit Investigation"
+
+	defaultOwner = "evergreen-ci"
+	defaultRepo  = "commit-queue-sandbox"
```

Comment:
What does the default owner and repo do for the installation token? Is it okay to hard-code these as the installation token source for production?

Reply:
We might want to move this to the admin page

Reply:
I think it should be, because evergreen is open source so it should be generic when it can be.

Reply:
(this can be done in a separate ticket though)